../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l1')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty="l1").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l2')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




b0 =  [-8.02999887]
Coefs =  [[ 0.12164685  0.03347113 -0.01044301  0.0059472  -0.00173991  0.07995459
   0.63944714  0.0144454 ]]
Logistic_reg_class_SCORE =  0.795
random_forest_class_SCORE =  0.985
KNN_class_SCORE =  0.815
SVM_class_SCORE =  0.7825
NB_class_SCORE =  0.7775

Traceback (most recent call last):
  File "diabetes-classifications.py", line 387, in <module>
    from plotly.plotly import iplot
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/plotly/plotly/__init__.py", line 4, in <module>
    _chart_studio_error("plotly")
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/_plotly_future_/__init__.py", line 49, in _chart_studio_error
    submodule=submodule
ImportError: 
The plotly.plotly module is deprecated,
please install the chart-studio package and use the
chart_studio.plotly module instead. 

[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'plotly.plotly'), ('new_fqn', 'chart_studio.plotly'), ('line_no', 387)])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




b0 =  [-8.02999887]
Coefs =  [[ 0.12164685  0.03347113 -0.01044301  0.0059472  -0.00173991  0.07995459
   0.63944714  0.0144454 ]]
Logistic_reg_class_SCORE =  0.795
random_forest_class_SCORE =  0.99
KNN_class_SCORE =  0.815
SVM_class_SCORE =  0.7825
NB_class_SCORE =  0.7775

[REPAIR EXEC TIME]: 111.23271346092224../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'dualbool')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l1')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="l1").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l2')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="l2").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'elasticnet')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="elasticnet").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'none')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="none").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'dualbool'), ('target_value', 'True')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool=True).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'dualbool'), ('target_value', 'False')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool=False).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'fit_intercept'), ('target_value', 'True')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',fit_intercept=True).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'fit_intercept'), ('target_value', 'False')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',fit_intercept=False).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'newton-cg')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="newton-cg").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'lbfgs')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="lbfgs").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'liblinear')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="liblinear").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'sag')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="sag").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'saga')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="saga").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'multi_class'), ('target_value', 'auto')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',multi_class="auto").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'multi_class'), ('target_value', 'ovr')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',multi_class="ovr").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'multi_class'), ('target_value', 'multinomial')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',multi_class="multinomial").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'warm_start'), ('target_value', 'True')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',warm_start=True).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'warm_start'), ('target_value', 'False')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',warm_start=False).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'dualbool&max_iter'), ('target_value', 'True&False')])  File "diabetes-classifications.py", line 159
    loj_reg = LogisticRegression(dualbool='l1',dualbool&max_iter="True&False").fit(x_train,y_train)
                                              ^
SyntaxError: keyword can't be an expression
../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'dualbool')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l1')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="l1").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l2')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="l2").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'elasticnet')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="elasticnet").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'none')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',penalty="none").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'dualbool'), ('target_value', 'True')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool=True).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'dualbool'), ('target_value', 'False')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool=False).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'fit_intercept'), ('target_value', 'True')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',fit_intercept=True).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'fit_intercept'), ('target_value', 'False')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',fit_intercept=False).fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'newton-cg')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="newton-cg").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'solver'), ('target_value', 'lbfgs')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1',solver="lbfgs").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[REPAIR EXEC TIME]: 36.152592182159424../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'dualbool')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'dualbool')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'tol')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'C')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'fit_intercept')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'intercept_scaling')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'class_weight')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'random_state')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'solver')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'max_iter')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'multi_class')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'verbose')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'warm_start')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'n_jobs')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'penalty'), ('to_name', 'l1_ratio')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_delete'), ('from_name', 'penalty'), ('to_name', '*')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(dualbool='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/utils/validation.py", line 73, in inner_f
    return f(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'dualbool'
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_rename'), ('from_name', 'dualbool'), ('to_name', 'penalty')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty='l1').fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l1')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




Traceback (most recent call last):
  File "diabetes-classifications.py", line 159, in <module>
    loj_reg = LogisticRegression(penalty="l1").fit(x_train,y_train)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 1304, in fit
    solver = _check_solver(self.solver, self.penalty, self.dual)
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/sklearn/linear_model/_logistic.py", line 443, in _check_solver
    "got %s penalty." % (solver, penalty))
ValueError: Solver lbfgs supports only 'l2' or 'none' penalties, got l1 penalty.
[Try Solution]: OrderedDict([('API', 'sklearn.linear_model.LogisticRegression'), ('action', 'arg_value_update'), ('arg_name', 'penalty'), ('target_value', 'l2')])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




b0 =  [-8.02999887]
Coefs =  [[ 0.12164685  0.03347113 -0.01044301  0.0059472  -0.00173991  0.07995459
   0.63944714  0.0144454 ]]
Logistic_reg_class_SCORE =  0.795
random_forest_class_SCORE =  0.9825
KNN_class_SCORE =  0.815
SVM_class_SCORE =  0.7825
NB_class_SCORE =  0.7775

Traceback (most recent call last):
  File "diabetes-classifications.py", line 387, in <module>
    from plotly.plotly import iplot
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/plotly/plotly/__init__.py", line 4, in <module>
    _chart_studio_error("plotly")
  File "/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/_plotly_future_/__init__.py", line 49, in _chart_studio_error
    submodule=submodule
ImportError: 
The plotly.plotly module is deprecated,
please install the chart-studio package and use the
chart_studio.plotly module instead. 

[Try Solution]: OrderedDict([('action', 'fqn'), ('old_fqn', 'plotly.plotly'), ('new_fqn', 'chart_studio.plotly'), ('line_no', 387)])../../../input/johndasilva_diabetes/diabetes.csv
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2000 entries, 0 to 1999
Data columns (total 9 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   Pregnancies               2000 non-null   int64  
 1   Glucose                   2000 non-null   int64  
 2   BloodPressure             2000 non-null   int64  
 3   SkinThickness             2000 non-null   int64  
 4   Insulin                   2000 non-null   int64  
 5   BMI                       2000 non-null   float64
 6   DiabetesPedigreeFunction  2000 non-null   float64
 7   Age                       2000 non-null   int64  
 8   Outcome                   2000 non-null   int64  
dtypes: float64(2), int64(7)
memory usage: 140.8 KB
satir ve sutun =  (2000, 9)
Boyut sayisi =  2
eleman sayisi =  18000




b0 =  [-8.02999887]
Coefs =  [[ 0.12164685  0.03347113 -0.01044301  0.0059472  -0.00173991  0.07995459
   0.63944714  0.0144454 ]]
Logistic_reg_class_SCORE =  0.795
random_forest_class_SCORE =  0.99
KNN_class_SCORE =  0.815
SVM_class_SCORE =  0.7825
NB_class_SCORE =  0.7775

[REPAIR EXEC TIME]: 154.2526023387909